### PR TITLE
Add affy/justrma module 

### PR DESCRIPTION
**Pull request by pinin4fjords**
This PR adds a module for basic affy array handling, wrapping the affy `justRMA()` function to generate matrices from collections of CEL files. This is modelled after previous R-based efforts such as [DESeq2](https://github.com/nf-core/modules/tree/master/modules/nf-core/deseq2) and [matrixfilter.](https://github.com/nf-core/modules/tree/master/modules/nf-core/custom/matrixfilter).

It required a couple of [upstream fixes](https://github.com/bioconda/bioconda-recipes/pull/39047), but we're there now. Singularity issues are just because the final upstream fix is too new, and the singularity magic hasn't happened yet. 

The associated [test data PR](https://github.com/nf-core/test-datasets/pull/738) needs to be merged before this, and pointed at accordingly.

## PR checklist

Closes https://github.com/nf-core/modules/issues/2789

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/affy/justrma/meta.yml
Comment (nvnieuwk) [inline modules/nf-core/affy/justrma/meta.yml:45]: This is not part of the output (probably left in by accident?)

**Review by nvnieuwk — APPROVED** [review]
LGTM apart from a small comment. You can merge if the test data has been added to the main repo and this module uses it :)

**Comment by nvnieuwk** [comment]
Also something seems wrong with the singularity container

**Comment by pinin4fjords** [comment]
> Also something seems wrong with the singularity container

@nvnieuwk This is just waiting for the container stuff to happen on the Galaxy end. I can hold off on the merge until it happens overnight, thanks for the review!

I've addressed the minor test data docs issue, and will re-point at that repo as soon as I can merge that PR.

File: modules/nf-core/affy/justrma/meta.yml
Comment (pinin4fjords) [reply modules/nf-core/affy/justrma/meta.yml]: Don't know how these things get past me- thanks!

Comment (nvnieuwk) [reply modules/nf-core/affy/justrma/meta.yml]: That's what reviews are for :p

**Pull request merged**
